.github/workflows/deploy-master.yml

```diff
@@ -70,6 +70,10 @@ jobs:
     - name: Build application
       run: |
         node common/scripts/install-run-rush.js rebuild -v
+      
+    - name: Build UI for production
```

Comment:
@JosephBagaric because there is no way to extract only origin-ui from building on `rebuild` command I have to first build it regular version and afterward build it for production. This is additional 80s, but this will happen only on master and release branches. Other solution is to override default `rush rebuild` command, which I personally don't like.

Reply:
That's fine.